packages/terra-status-view/src/StatusView.module.scss

```diff
@@ -46,7 +46,8 @@
   .glyph {
     padding-bottom: 0.714rem;
 
-    > svg {
+    > svg,
```

Comment:
this is out of scope and probably a breaking change, but i wonder if we can eventually remove the `svg` selector. currently, it only applies if a consumer passes an svg via the `customGlyph` prop. its probably better if we let consumers handle their own svg styles. 

tho, i think our plan is to eventually remove custom functionality from the status-view in some matter. https://github.com/cerner/terra-core/issues/3012

Reply:
I think you are right that we can probably remove this when we remove the support for custom svgs. It's prob fine for now to leave as is